### ISSUE TITLE
GPU plugin foilage glitches with RuneLite 1.10.11.2

### ISSUE THREAD

**Issue by JacobSvenningsen** [runelite/runelite#17000]
### Type

GPU

### Operating System

Linux

### OS Version

Debian Sid

### Bug description

GPU plugin has graphical glitches with RuneLite version 1.10.11.2 if Compute Shaders is enabled. It's quite noticable around farming patches.
It worked yesterday and just a few hours earlier today as well, so I assume it's a regression between 1.10.11.2 and 1.10.11.1. The attached logs hints at the same, since the first entry lists 1.10.11.1 as well.

Enabled:
![image](https://github.com/runelite/runelite/assets/5268510/62e4119b-b74c-4677-a721-f9e76dc4c38d)

Disabled:
![image](https://github.com/runelite/runelite/assets/5268510/313068af-3733-42e3-b694-0d102d35d053)

### RuneLite version

RuneLite version: 1.10.11.2
Launcher version: 2.6.7

### Logs

[launcher.log](https://github.com/runelite/runelite/files/12551278/launcher.log)
[client.log](https://github.com/runelite/runelite/files/12551279/client.log)

**Comment by Adam-** [comment]
This is certainly caused by the changes in 1.10.11.2 as I moved a lot of additional logic to shaders. The shaders seem to be working correctly on other's systems so far. Can you get the output of glxinfo?

**Comment by JacobSvenningsen** [comment]
attached glxinfo
[glxinfo.txt](https://github.com/runelite/runelite/files/12551382/glxinfo.txt)

**Comment by JacobSvenningsen** [comment]
Since I also got a laptop, I tried downloading RuneLite on it as well.
The issue is present there as well.
OS: Debian 12 (Bookworm)

Laptop uses an integrated Intel GPU.

Client, launcher and glxinfo logs attached

[client.log](https://github.com/runelite/runelite/files/12551971/client.log)
[glxinfo.txt](https://github.com/runelite/runelite/files/12551972/glxinfo.txt)
[launcher.log](https://github.com/runelite/runelite/files/12551976/launcher.log)

**Comment by Adam-** [comment]
Can you try changing both of these https://github.com/runelite/runelite/blob/0e3393d3ff7b36cbd88ba01ddb58abb8eca296a8/runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java#L1529-L1530 to GL_NEAREST?

**Comment by JacobSvenningsen** [comment]
Never compiled a bigger java project before, nevermind a Maven one, and I'm afraid the build instructions assumes too many things.

I did try, but failed to compile tag [runelite-parent-1.10.11.2](https://github.com/runelite/runelite/releases/tag/runelite-parent-1.10.11.2) without any modifications.

By "Open the project in your IDE as a Maven project", I installed IntelliJ Community Edition, then opened the root of the git repository inside the IDE. Considering the right-hand side of the IDE has a Maven button, I guess it's already opened as a Maven project?
![image](https://github.com/runelite/runelite/assets/5268510/cc5e0e0a-01c7-4e06-9e9a-5ea1e6511522)

By "build the root module", I guess you mean the root of the tree that's just called "RuneLite"?
I chose to select the option "Generate Sources and Update Folders", followed by `mvn compile -X`. Though it fails with the error when it compiles RuneLite Client.

```
[ERROR] Failed to parse plugin descriptor for net.runelite:script-assembler-plugin:1.10.11.2 (/home/hu/Sources/runelite/runelite-script-assembler-plugin/target/classes): No plugin descriptor found at META-INF/maven/plugin.xml -> [Help 1]
org.apache.maven.plugin.PluginDescriptorParsingException: Failed to parse plugin descriptor for net.runelite:script-assembler-plugin:1.10.11.2 (/home/hu/Sources/runelite/runelite-script-assembler-plugin/target/classes): No plugin descriptor found at META-INF/maven/plugin.xml
```

Is there some sort of dependency that Maven doesn't fetch? Right-clicking the root and selecting "analyse dependencies" just lists nothing
